ntbs-service/Pages/Shared/_Layout.cshtml

```diff
@@ -9,12 +9,17 @@
 <body>
     <div id="app">
         <div>
+            @RenderSection("SkipLink", false)
+            @* nhs-main-wrapper renders <main id="maincontent"></main> *@
```

Comment:
do we need this line for future debugging purposes?

Reply:
It is more for context, as the `href` is currently to a value which we don't set ourselves. Unless you feel strongly about it I think I would rather leave it in?